syntax = "proto3";

service SeatingCall {
  rpc AddChairToSeatingArrangement(AddChairToSeatingArrangementRequest) returns (AddChairToSeatingArrangementReply){}
  rpc RemoveChairFromSeatingArrangement(RemoveChairFromSeatingArrangementRequest) returns (RemoveChairFromSeatingArrangementReply){}
}

message AddChairToSeatingArrangementRequest {
  int32 class_id = 1;
  string arrangement = 2;
}

message AddChairToSeatingArrangementReply {
  optional string message = 1;
  optional string error = 2;
  int32 status_code = 3;
}

message RemoveChairFromSeatingArrangementRequest {
  string chair_ids = 1;
}

message RemoveChairFromSeatingArrangementReply {
  optional string message = 1;
  optional string error = 2;
  int32 status_code = 3;
}